LondonBritishLibrary/orient/BLorient4930.xml

```diff
@@ -0,0 +1,400 @@
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.5">
+                                        <locus from="130r" to="131r"/>
+                                        <title ref="LIT1828Mahale#Hezekiah"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.6">
+                                        <locus from="131r" to="132r"/>
+                                        <title ref="LIT1828Mahale#Manasseh"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.7">
+                                        <locus target="#132r"/>
+                                        <title ref="LIT1828Mahale#Jonah"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.8">
+                                        <locus from="132r" to="133v"/>
+                                        <title ref="LIT1828Mahale#ThreeYouths1"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.9">
+                                        <locus target="#133v"/>
+                                        <title ref="LIT1828Mahale#ThreeYouths2"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.10">
+                                        <locus from="133v" to="134v"/>
+                                        <title ref="LIT1828Mahale#ThreeYouths3"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.11">
+                                        <locus from="134v" to="136r"/>
+                                        <title ref="LIT1828Mahale#Habakkuk"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.12">
+                                        <locus from="136r" to="136v"/>
+                                        <title ref="LIT1828Mahale#Isaiah"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.13">
+                                        <locus from="136v" to="137r"/>
+                                        <title ref="LIT1828Mahale#Mary"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.14">
+                                        <locus from="137r" to="137v"/>
+                                        <title ref="LIT1828Mahale#Zachariah"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.15">
+                                        <locus target="#137v"/>
+                                        <title ref="LIT1828Mahale#Simeon"/>
+                                    </msItem>
```

Comment:
should the invocation go somewhere here? if it is the same hand and same unit, then we follow the text order in the manuscript

Reply:
If I put ms_i3 into ms_i1, should I do the same as well with ms_i2 and ms_i4 and perhaps even with ms_i5, ms_i6 and ms_i7?